tests/jfrogclient_test.go

```diff
@@ -43,6 +43,8 @@ func InitServiceManagers() {
 	createArtifactoryReplicationGetManager()
 	createArtifactoryReplicationDeleteManager()
 	createArtifactoryPermissionTargetManager()
+	createArtifactoryUsersManager()
```

Comment:
createArtifactoryUsersManager --> createArtifactoryUserManager
(for consistency with the other APIs).